examples/11-kitchen-sink/.brigade/brigade.ts

```diff
@@ -17,8 +17,9 @@ events.on("brigade.sh/cli", "exec", async event => {
 
   const job2Name = newJobName()
   let job2 = new Job(job2Name, "alpine:latest", event)
-  job2.primaryContainer.command = ["echo"]
-  job2.primaryContainer.arguments = [`Hello, ${job2Name}!`]
+  job2.primaryContainer.command = ["sleep"]
```

Comment:
Were the changes in this file intended for this PR?

Reply:
Brilliant catch. No. Not intended. This snuck in somehow because I was working on timeout fix stuff in parallel.

Will fix.

Reply:
Fixed